setup.py

```diff
@@ -55,6 +55,8 @@ def run(self):
 
 install_requires = [
     "alembic>=0.6.6",
+    "amqp>=1.4.5,<2.0",
```

Comment:
`amqp` should be optional extra `"rabbitmq": "amqp>=1.4.5,<2.0",`. Why do you need to bind the maximum version?